pyproject.toml

```diff
@@ -54,27 +55,36 @@ nbconvert = ">=5.4.0"
 nbformat = ">=4.4.0"
 nbsphinx = {version = "*", optional = true}
 nbsphinx-link = {version = "*", optional = true}
-networkx = ">=2.5"
+networkx = "==3.1"
 ngs-tools = ">=1.5.6"
 numba = ">=0.51.0"
-numpy = ">=1.19.5,<1.22"
+numpy = ">=1.22"
+opencv-python = {version = ">=4.5.4.60", optional = true}
 pandas = ">=1.1.4"
 parameterized = "*"
 plotly = ">=5.0.0"
+poisson-disc = {version = ">=0.2.1", optional = true}
 pre-commit = {version = ">=2.7.1", optional = true}
 pydata-sphinx-theme = {version = ">=0.4.3", optional = true}
 pysam = ">=0.14.1"
 pyseq-align = ">=1.0.2"
 pytest = {version = ">=4.4", optional = true}
 python = ">=3.7,<4.0"
+pyvista = {version = "=0.41.0", optional = true}
```

Comment:
I've fixed version for now since there is a bug in 0.41.1. Can change once this is resolved